bin/deploy_automation/_global_vars.sh

```diff
@@ -8,7 +8,7 @@ export GITHUB_TOKEN; GITHUB_TOKEN=${GITHUB_TOKEN:-$(cut -f 3 -d : ~/.git-credent
 export GITHUB_REMOTE_ORIGIN; GITHUB_REMOTE_ORIGIN=$(git remote get-url origin | sed -E "s/^(https|git)(:\/\/|@)([^\/:]+)[\/:]([^\/:]+)\/(.+).git$/\\4\/\\5/")
 export GITHUB_REPOSITORY_API="https://api.github.com/repos/${GITHUB_REMOTE_ORIGIN}"
 export GITHUB_REPOSITORY_URL="https://github.com/${GITHUB_REMOTE_ORIGIN}"
-export DOCKER_REPOSITORY_NAME="$GITHUB_REMOTE_ORIGIN"
+export ECR_REPOSITORY_NAME; ECR_REPOSITORY_NAME=$(cut -f 2 -d / <<<"$GITHUB_REMOTE_ORIGIN")
```

Comment:
I might be missing something, but doesn't the ECR repo need to be something like `blah.dkr.ecr.us-west-2.amazonaws.com` ?

Reply:
The command I'm using actually requires the repository name, as in: 
`aws ecr describe-images --repository-name "idseq-web" --image-ids imageTag="sha-630aeb3c"`